### DISCUSSION TITLE
Financial Support / Donations

### DISCUSSION THREAD

**Discussion by dagnum** [SignalK/SensESP#709] [General]
Hello!  I started with SigkSense and moved over to SensEsp when the time came!  I think what you are doing just like Signalk is amazing.  I would like to know if you have anyway we can support you financially.  I would be happy to send you $5 a month for what you are doing and I am sure there are others that would do the same.  I am still working on getting basics running perfectly smooth but really hope to help others install on their boats too.  Do you have anywhere for us users to send you money?

**Comment by tkurki** [comment]
Really appreciate your interest - Signal K contributions are at https://opencollective.com/signalk.

**Reply by dagnum** [reply]
Oh awesome, I thought you were independent of SignalK!  I am already on the $5 a month plan!  Thank you!
